src/main/java/org/opentripplanner/model/SimpleTransfer.java

```diff
@@ -37,38 +39,32 @@ public String getName() {
     }
 
     public double getDistanceMeters() {
-        return edges.stream().mapToDouble(Edge::getDistanceMeters).sum();
+        return distanceMeters;
     }
 
-    /**
-     * The distance to walk adjusted for elevation and obstacles. This is used together
-     * with the walking speed to find the actual walking transfer time. This plus
-     * {@link #getDistanceIndependentTime()} is used to calculate the actual-transfer-time
-     * given a walking speed.
-     * <p>
-     * Unit: meters. Default: 0.
-     * @see Edge#getEffectiveWalkDistance()
-     */
-    public double getEffectiveWalkDistance(){
-    	return this.effectiveWalkDistance;
-    }
+    public List<Edge> getEdges() { return this.edges; }
 
-    /**
-     * This is the transfer time(duration) spent NOT moving like time in in elevators, escalators
-     * and waiting on read light when crossing a street. This is used together with
-     * {@link #getEffectiveWalkDistance()} to calculate the actual-transfer-time.
-     * <p>
-     * Unit: seconds. Default: 0.
-     * @see Edge#getDistanceIndependentTime()
-     */
-    public int getDistanceIndependentTime() {
-        return distanceIndependentTime;
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) { return true; }
+        if (o == null || getClass() != o.getClass()) { return false; }
+        final SimpleTransfer that = (SimpleTransfer) o;
+        return Double.compare(that.distanceMeters, distanceMeters) == 0 && from.equals(
+                that.from) && to.equals(that.to) && Objects.equals(edges, that.edges);
     }
```

Comment:
All internal model classes implementing eq/hc must document why they do so. The "rule" is to use the Object::hc/eq for best performance. All these objects are created at import time and the import modules (graph builder) is responsible for deduplication. We might have to consider this strategy again IF we start creating objects as part of the RT updates.

Reply:
The `hashCode()/equals()` implementation has been removed, since it is not needed.